README.md

```diff
@@ -1,2 +1,3 @@
 # Git & Github training exercises
 Git & Github training exercises for when interacting with a repository that is not yours. 
+This link is the link https://www.youtube.com/watch?v=2X1FjFqIwIY to learn about contributing on github
```

Comment:
Hi @oyenuga17 Great job submitting your 1st PR. but you didn't finish the task. Please the following:
- Add the section heading before the statement you wrote.
- Convert the link to mark down. Let me help you below:
Copy and paste this line instead:
````
## Change this to heading name
Watch this video to learn how to add content to a repo that is not yours, through the Github UI: [GADS project gallery Github tutorial](https://www.youtube.com/watch?v=2X1FjFqIwIY)
````

Reply:
Cool! Great job. This is perfect!